tests/integration/test_api_literature.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
```

Comment:
Ah ok, was saying because our tests are named at the moment:

`test_<whatever>_views.py`

So thought might be better for API tests to be named:

``test_<whatever>_api.py`